mono/utils/Makefile.am

```diff
@@ -27,6 +27,9 @@ monoutils_sources = \
 	mono-dl-darwin.c	\
 	mono-dl-posix.c		\
 	mono-dl.h		\
+	mono-log-windows.c	\
+	mono-log-common.c	\
+	mono-log-posix.c	\
```

Comment:
These new files need to be added to https://github.com/mono/mono/blob/master/msvc/libmonoutils.vcxproj too (MSVC build is broken for the PR currently).